src/main/java/org/broadinstitute/hellbender/utils/read/GATKReadToBDGAlignmentRecordConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.function.Function;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import org.bdgenomics.adam.converters.SAMRecordConverter;
+import org.bdgenomics.adam.models.SequenceDictionary;
+import org.bdgenomics.adam.models.RecordGroupDictionary;
+
+
+import java.io.Serializable;
+
+/**
+ * Converts a GATKRead to a BDG AlignmentRecord
+ */
+public class GATKReadToBDGAlignmentRecordConverter implements Function<GATKRead, AlignmentRecord> {
```

Comment:
Gotcha. What's the proper way to deal with the header then?  Do you prefer that the static `convert` method also take in the sequence dictionary etc?